bzl/plaidml.bzl

```diff
@@ -45,10 +45,7 @@ def plaidml_cc_test(copts = [], deps = (), linkopts = [], **kwargs):
     native.cc_test(
         copts = PLAIDML_COPTS + copts,
         deps = deps + [str(Label("//testing:gtest_main"))],
-        linkstatic = select({
-            "@com_intel_plaidml//toolchain:linux_x86_64": 1,
-            "//conditions:default": None,
-        }),
+        linkstatic = 1,
```

Comment:
Is this still required or was it just for testing/debugging?

I'm actually curious why we need this (just for my understanding)

Reply:
We need this to link in the tbb into the test executables. I dumped the Linux ones (nm is your friend :) ) and the note_affinity (a  tbb method) was removed by the linker since it was not called internally by anything - just the static initializer code of tbb.
Using alwayslink flag on the TBB and the xmm did not resolve the issues. Doubler checked dumping things also and the always link will keep the methods in the static library (they are always there anyway), but it will still remove them from the final executable when it links the library in. The linkstatic flags keeps the static symbols that are used by static initializers as well. (From what I can tell running dumps and analyzing things).

Reply:
The downside is that it makes our test binaries very large especially when building in debug mode, and so it causes builds to take on the order of minutes to run tests in a quick cycle. I'm OK with leaving this in for now, but I'd like to revisit this to see if we can find a better solution.